Multisig/UI/UI Library/Views/OwnerActionView.swift

```diff
@@ -30,7 +30,7 @@ class OwnerActionView: UINibView {
         case .addingOwner:
             actionTag.set(title: "Adding owner", style: .footnote2, backgroundColor: .backgroundPositive, textColor: .primary)
         case .removingOwner:
-            actionTag.set(title: "Removing owner", style: .footnote2, backgroundColor: .backgroundPrimary, textColor: .labelSecondary)
+            actionTag.set(title: "Removing owner", style: .footnote2, backgroundColor: .backgroundError, textColor: .error)
```

Comment:
this is wrong: the designs are different for the 'removing owner' when this part is used in the "replace" and when it is used in the "remove" use cases. This change introduces a mistake for the "replace owner" design UI